drivers/espi/espi_mchp_xec_v2.c

```diff
@@ -132,18 +132,18 @@ static const struct xec_signal vw_tbl[] = {
 	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SCI, vw_sci_n),
 	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_DNX_ACK, vw_dnx_ack),
 	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SUS_ACK, vw_sus_ack_n),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_0, vw_c2t_gpio_0),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_1, vw_c2t_gpio_1),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_2, vw_c2t_gpio_2),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_3, vw_c2t_gpio_3),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_4, vw_c2t_gpio_4),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_5, vw_c2t_gpio_5),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_6, vw_c2t_gpio_6),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_7, vw_c2t_gpio_7),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_OCB_0, vw_c2t_gpio_8),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_OCB_1, vw_c2t_gpio_9),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_OCB_2, vw_c2t_gpio_10),
-	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_OCB_3, vw_c2t_gpio_11),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_0, vw_t2c_gpio_0),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_1, vw_t2c_gpio_1),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_2, vw_t2c_gpio_2),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_3, vw_t2c_gpio_3),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_4, vw_t2c_gpio_4),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_5, vw_t2c_gpio_5),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_6, vw_t2c_gpio_6),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_SLV_GPIO_7, vw_t2c_gpio_7),
+	MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_OCB_0, vw_t2c_gpio_8),
```

Comment:
with the new change ESPI_VWIRE_SIGNAL_OCB_0 = ESPI_VWIRE_SIGNAL_SLV_GPIO_0
should these be ESPI_VWIRE_SIGNAL_SLV_GPIO_8 and above?

Reply:
this table is generic, doesn't know of the intended mapping, so has all the vw gpios in the enum `espi_vwire_signal`

Reply:
ACK